pkg/controller/kabaneroplatform/landing.go

```diff
@@ -203,18 +204,18 @@ func cleanupLandingPage(k *kabanerov1alpha2.Kabanero, c client.Client) error {
 		return err
 	}
 
-	m, err := mf.FromReader(strings.NewReader(s), c)
+	mOrig, err := mf.ManifestFrom(mf.Reader(strings.NewReader(s)), mf.UseClient(mfc.NewClient(c)))
 	if err != nil {
 		return err
 	}
 
 	transforms := []mf.Transformer{mf.InjectOwner(k), mf.InjectNamespace(k.GetNamespace())}
-	err = m.Transform(transforms...)
+	m, err := mOrig.Transform(transforms...)
 	if err != nil {
 		return err
 	}
 
-	err = m.DeleteAll()
+	err = m.Delete()
 	if err != nil {
 		if !apierrors.IsNotFound(err) {
```

Comment:
This is redundant. The default behavior of `m.Delete()` will "eat" the `NotFound` error. There is an option to override that behavior, but I think most folks just want the manifest resources gone.

Reply:
Great - I like that default behavior :smile: